pnpm-lock.yaml

```diff
@@ -9005,6 +9006,7 @@ importers:
       '@fluidframework/driver-definitions': link:../../common/driver-definitions
       '@fluidframework/id-compressor': link:../id-compressor
       '@fluidframework/protocol-definitions': 3.2.0
+      '@fluidframework/telemetry-utils': link:../../utils/telemetry-utils
```

Comment:
You can/should revert the changes to this file besides this and the one above.  It's fine to bump those dependencies, but I'd strongly recommend doing it in a separate PR for clarity.